newIDE/app/src/MainFrame/index.js

```diff
@@ -459,6 +459,26 @@ class MainFrame extends React.Component<Props, State> {
       .then(() => this.setState({ loadingProject: false }));
   };
 
+  closeApp = (): void => {
+    Window.quit();
+  };
```

Comment:
Don't make a method if it's not used outside and just calling a function. Prefer just writing Window.quit, which is very clear by itself.

Reply:
Okay, correcting this.